src/utils/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const baseURL = "http://nc-news-api-hjp3.onrender.com/api";
```

Comment:
I think this should be https: ?? This is causing an error in the browser

![image](https://github.com/prgf87/fe-project-nc-news/assets/72257311/6935e724-6a10-44b6-9b49-6f57837b4141)